charts/chartmuseum/values.yaml

```diff
@@ -0,0 +1,358 @@
+    # of: AES256 or KMS
+    STORAGE_ALIBABA_SSE:
+    # s3 bucket to store charts for amazon storage backend
+    STORAGE_AMAZON_BUCKET:
+    # prefix to store charts for amazon storage backend
+    STORAGE_AMAZON_PREFIX:
+    # region of s3 bucket to store charts
+    STORAGE_AMAZON_REGION:
+    # alternative s3 endpoint
+    STORAGE_AMAZON_ENDPOINT:
+    # server side encryption algorithm
+    STORAGE_AMAZON_SSE:
+    # gcs bucket to store charts for google storage backend
+    STORAGE_GOOGLE_BUCKET:
+    # prefix to store charts for google storage backend
+    STORAGE_GOOGLE_PREFIX:
+    # container to store charts for microsoft storage backend
+    STORAGE_MICROSOFT_CONTAINER:
+    # prefix to store charts for microsoft storage backend
+    STORAGE_MICROSOFT_PREFIX:
+    # container to store charts for openstack storage backend
+    STORAGE_OPENSTACK_CONTAINER:
+    # prefix to store charts for openstack storage backend
+    STORAGE_OPENSTACK_PREFIX:
+    # region of openstack container
+    STORAGE_OPENSTACK_REGION:
+    # path to a CA cert bundle for your openstack endpoint
+    STORAGE_OPENSTACK_CACERT:
+    # compartment id for for oracle storage backend
+    STORAGE_ORACLE_COMPARTMENTID:
+    # oci bucket to store charts for oracle storage backend
+    STORAGE_ORACLE_BUCKET:
+    # prefix to store charts for oracle storage backend
+    STORAGE_ORACLE_PREFIX:
+    # form field which will be queried for the chart file content
+    CHART_POST_FORM_FIELD_NAME: chart
+    # form field which will be queried for the provenance file content
+    PROV_POST_FORM_FIELD_NAME: prov
+    # levels of nested repos for multitenancy. The default depth is 0 (singletenant server)
+    DEPTH: 0
+    # show debug messages
+    DEBUG: false
+    # output structured logs as json
+    LOG_JSON: true
+    # disable use of index-cache.yaml
+    DISABLE_STATEFILES: false
+    # enable Prometheus metrics
+    ENABLE_METRICS: false
+    # disable all routes prefixed with /api
+    DISABLE_API: true
+    # allow chart versions to be re-uploaded
+    ALLOW_OVERWRITE: false
+    # absolute url for .tgzs in index.yaml
+    CHART_URL:
+    # allow anonymous GET operations when auth is used
+    AUTH_ANONYMOUS_GET: false
+    # sets the base context path
+    CONTEXT_PATH:
+    # parallel scan limit for the repo indexer
+    INDEX_LIMIT: 0
+    # cache store, can be one of: redis (leave blank for inmemory cache)
+    CACHE:
+    # address of Redis service (host:port)
+    CACHE_REDIS_ADDR:
+    # Redis database to be selected after connect
+    CACHE_REDIS_DB: 0
+    # enable bearer auth
+    BEARER_AUTH: false
+    # auth realm used for bearer auth
+    AUTH_REALM:
+    # auth service used for bearer auth
+    AUTH_SERVICE:
+        # username for basic http authentication
+  field: {}
+    # POD_IP: status.podIP
+  secret:
+    # username for basic http authentication
+    BASIC_AUTH_USER: admin
```

Comment:
compare with origin, line 104 and line 106 changed.
```diff
   secret:
     # username for basic http authentication
+    BASIC_AUTH_USER: admin
-    BASIC_AUTH_USER:
     # password for basic http authentication
+    BASIC_AUTH_PASS: password
-    BASIC_AUTH_PASS:
     # GCP service account json file

```
I guess @bjwswang https://github.com/kubebb/components/pull/69#issuecomment-1647601621 wants to keep the no username password by default, and just add a new example that uses basic authentication...
> add a examplecomponentplan for chartmuseum wich enables basic auth

Reply:
@zqq454224016 @Abirdcfly  that's right.  maybe two examples : one withouth any auth and one with basic auth

Reply:
I dont get that. We would like to add auth args when we use func repoadd . So do we need to add auth args in componentplan.yaml, or add secret.yaml when deploying componentplan.yaml?